verification/example_config.toml

```diff
@@ -6,3 +6,6 @@ addr = "0.0.0.0:8043"
 api_url = "https://sourcify.dev/server/"
 verification_attempts = 3
 request_timeout = 10
+
+[compiler]
+compilers_list_url = "https://binaries.soliditylang.org/linux-amd64/"
```

Comment:
And does not specifying the actual file seems a little confusing. What if we use the full path (including `/list.json` segment) allowing any file name to be used as a list? And if the `path` has no schema prefix, we will use the url without that last segment